src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -163,7 +164,7 @@ const ChatItemMyTextV2 = ({
           </View>
         )}
         {attachments.length > 0 && <ChatItemAttachment attachments={attachments} />}
-        {attachments.length <= 0 && <Text style={styles.text}>{message}</Text>}
+        {attachments.length <= 0 && <LinkableText style={styles.text} text={message} />}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the `Text` component with the new `LinkableText` component seems appropriate, given the context provided. However, it's important to ensure that the `LinkableText` component has been thoroughly tested and handles all edge cases that the `Text` component did. Also, make sure that the `LinkableText` component does not introduce any performance issues due to additional processing for link detection.
```diff
- {attachments.length <= 0 && <Text style={styles.text}>{message}</Text>}
+ {attachments.length <= 0 && <LinkableText style={styles.text} text={message} />}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->